.eslintrc.json

```diff
@@ -109,8 +110,10 @@
   },
   "overrides": [
     {
+      "extends": ["plugin:testing-library/react"],
       "files": ["tests/**/*.ts", "tests/**/*.tsx"],
       "rules": {
+        "testing-library/no-manual-cleanup": "off",
```

Comment:
Added as seems like cleanup is not done automatically, and cleanup added in `setup.ts` is still needed.

Reply:
Automatic cleanup is not enabled as current tests configuration does not have `globals`. to `true`. More on that: https://github.com/testing-library/vue-testing-library/issues/296.

I would leave this rule turned off rather than enabling globals.